packages/pn-commons/src/types/Notifications.ts

```diff
@@ -27,4 +27,5 @@ export interface GetNotificationsParams {
   size?: number;
   nextPagesKey?: string;
   iunMatch?: string;
+  isDelegatedPage?: boolean;
```

Comment:
I shoul not include this flag into this model, because it has a different meaning. I should pass it as another parameter in the action